src/semantic-ui/site/elements/container.overrides

```diff
@@ -268,3 +268,16 @@ Container Overrides - REACT-INVENIO-APP-ILS
     }
   }
 }
+
+.ui.container.link-like {
+    color: #4183C4 !important;
+    cursor: pointer !important;
+}
+
+.ui.container.top-spaced {
```

Comment:
did you check how was the space created for the View ALL link on the main page? We should avoid the custom CSS if possible

Reply:
It doesn't have a margin (the margin is due to the title it has on top).

Reply:
yes because the negative space is there created by `<Divider>` not CSS class. That is why I suggested for you too look how it is done there.